code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -46,14 +46,17 @@ GLOBAL_LIST_EMPTY(damage_icon_parts)
 
 //Add an entry to overlays, assuming it exists
 /mob/living/carbon/human/proc/apply_layer(cache_index)
-	if((. = overlays_standing[cache_index]))
-		add_overlay(.)
+	var/overlay_slot = overlays_standing[cache_index]
+	. = overlay_slot
+	if(overlay_slot)
+		// `add_overlay()` will mutate lists you pass it.
+		add_overlay(islist(overlay_slot) ? overlay_slot:Copy() : overlay_slot)
```

Comment:
why overlay_slot:Copy() and not overlay_slot.Copy()?

Reply:
Per comment on discord, just because the var is untyped -- the var could be made a list, but the slot code is doing type abuse so either way it's unsafe. Might as well be explicit about the type unsafety.